lib/msal-node/docs/caching.md

```diff
@@ -0,0 +1,109 @@
+# Token caching in MSAL Node
+
+When MSAL Node acquires a token, it caches it in memory for future usage. MSAL Node manages the token lifetime and refreshing for you. APIs like `acquireTokenSilent()` retrieves the access tokens from the cache for a given account:
+
+```javascript
+const msal = require('@azure/msal-node');
+
+// Create msal application object
+const cca = new msal.ConfidentialClientApplication({
```

Comment:
Please add a comment on where a user can go to get their clientId, tenant info, and clientSecret.  Something that hints what the format would look like for the actual data might be useful for users as well.
clientId:  "your client ID goes here",  // Example "00000..."
Gives the reader an idea whether they have the right format for the items in question.

Reply:
added!

Reply:
Where was this added?

Reply:
weird.. @peterzenz see [latest commit](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/4003/commits/c47484339c866520193e2d325d38875d17f43ac8)

Reply:
Helpful for the home account id below, but  I don't see anything for clientID, tenant info and clientSecret.  Am I missing something?

Reply:
My bad, i was fixated on home account id and missed your point. Updated now.